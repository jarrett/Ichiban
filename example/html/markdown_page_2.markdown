## Markdown Page 2

This is a paragraph.

This is another.