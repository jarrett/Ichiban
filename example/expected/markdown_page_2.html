<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <h2 id="markdown-page-2">Markdown Page 2</h2>

<p>This is a paragraph.</p>

<p>This is another.</p>

  </body>
</html>